hazelcast/src/main/java/com/hazelcast/cluster/impl/ClusterClockImpl.java

```diff
@@ -24,6 +24,7 @@
 
     private final ILogger logger;
     private volatile long clusterTimeDiff = Long.MAX_VALUE;
+    private volatile long clusterStartTime = Long.MIN_VALUE;
```

Comment:
It is used for controlling whether the value is set before or not. If it is not MIN_VALUE, I do not update the value.